docs/documentation/quickstarts/nextjs.mdx

```diff
@@ -67,6 +67,182 @@ It will ask you a few questions
 2. Enter your development API key. Enter the key you copied earlier.
 3. Enter a unique ID for your endpoint (you can just use the default by hitting enter)
 
+
+### Add the `TriggerProvider` component
+
+The [TriggerProvider](/sdk/react/triggerprovider) component is a React Context Provider that will make the Trigger API client available to all child components.
+
+Generally you'll want to add this to the root of your app, so that it's available everywhere. However, you can add it lower in the hierarchy but it must be above any of the hooks.
+
+```tsx app/layout.tsx
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <TriggerProvider publicApiKey={process.env.NEXT_PUBLIC_TRIGGER_API_KEY!}>
```

Comment:
We need to add this environment variable during `init` as well. You'll need to add `pkApiKey` to the `WhoamiResponseSchema` here: https://github.com/triggerdotdev/trigger.dev/blob/302bd02ff16bdba5dce811e93c605d0f832c7920/packages/cli/src/utils/triggerApi.ts#L30

The `/api/v1/whoami` endpoint already returns the `pkApiKey` so no changes will need to be made on the API side.

Reply:
I'll fix this ASAP.